relay_messages_test.go

```diff
@@ -215,6 +222,38 @@ func withLazyCallResCombinations(f func(cr testCallRes)) {
 	}
 }
 
+func newCallResFrame(tb testing.TB, p testCallResParams) *Frame {
+	f := NewFrame(p.payloadSize)
+	f.Header = FrameHeader{
+		// size is set below after construction of payload
+		messageType: messageTypeCallRes,
+		ID:          0xDEADBEEF,
+	}
+	require.NoError(tb, f.Header.write(typed.NewWriteBuffer(f.headerBuffer)))
```

Comment:
We're writing the header here, but then we update the size in the header later. See how this is done for call req,
```
	// Set the size in the header and write out the header after we know the payload contents.
	defer func() {
		fh.size = FrameHeaderSize + uint16(len(f.Payload))
		f.Header = fh
		require.NoError(t, fh.write(typed.NewWriteBuffer(f.headerBuffer)), "failed to write header")
	}()
```

Reply:
got it. will change for consistency